### DISCUSSION TITLE
Appreciation post

### DISCUSSION THREAD

**Discussion by bl4zee1g** [mywalkb/LSPosed_mod#77] [General]
you saved me! I physically can't use Snapchat without Snapenhance anymore, same goes for discord without bunny! I really hope you won't face any harrassment like it happened with the LSPosed devs, history shall not repeat itself this time around! Much love from Switzerland :3

**Comment by mywalkb** [comment]
I'm glad to read that you like it. I don't follow social media much, so I don't care much about what other people say. People always find an excuse to waste time.
